About.html

```diff
@@ -18,7 +18,7 @@
   <header id="HeaderM" class="HeaderMobile">
```

Comment:
## [OPTIONAL]

- [ ] Also, please consider a background color to you image container and some opacity as you can see to the original design :point_down: 

``` 
    background-color:  #ec5242;
    opacity: 0.5;
```
Yours  | Figma
------------- | -------------
![bb1](https://user-images.githubusercontent.com/57562869/152615632-d8d7cafa-0d9d-4038-80a9-74cbddc46bab.png)| ![gg2](https://user-images.githubusercontent.com/57562869/152615645-8f7adb86-39f9-43dc-8afa-580c8485e13b.png)